superset-frontend/src/dashboard/components/DashboardBuilder/DashboardBuilder.tsx

```diff
@@ -69,10 +69,11 @@ const StyledDiv = styled.div`
   flex: 1;
 `;
 
+// @z-index-above-dashboard-charts + 1 = 11
```

Comment:
We should probably start a z-index registry somewhere. The comment is appreciated in the meantime.